ui/webui/src/components/storage/DiskEncryption.jsx

```diff
@@ -91,6 +91,34 @@ const getStrengthLevels = (minQualility, isStrict) => {
     return levels;
 };
 
+const rules = [
+    {
+        id: "length",
+        text: (policy) => cockpit.format(_("Must be at least $0 characters"), policy["min-length"].v),
+        check: (policy, password) => password.length >= policy["min-length"].v,
+        isWarning: false,
+    },
+    {
+        id: "ascii",
+        text: (policy) => _("The passphrase you have provided contains non-ASCII characters. You may not be able to switch between keyboard layouts when typing it."),
+        check: (policy, password) => password.length > 0 && /^[\x20-\x7F]*$/.test(password),
+        isWarning: true,
+    },
+];
+
+const getRuleResults = (rules, policy, password) => {
+    return rules.map(rule => {
+        return {
+            id: rule.id,
+            text: rule.text(policy, password),
+            satisfied: password.length > 0 ? rule.check(policy, password) : null,
```

Comment:
satisfied is a boolean like the isWarning variable. For consistency purposes, let's pick one and use either the pattern 'isMyBoolVariable' or 'myBoolVariable'.

Reply:
Updated